lib/pages/rubsignin/widgets/signin_button.dart

```diff
@@ -25,7 +25,7 @@ class LoginInButton extends StatelessWidget {
         child: Center(
           child: Text(
             buttonText,
-            style: themeData.textTheme.headline1!.copyWith(
+            style: themeData.textTheme.displayLarge!.copyWith(
```

Comment:
Does `textTheme.displayLarge` default to `textTheme.headline1`? Because it could lead to `null` and therefore in another font style.

Reply:
According to the doc, it does.
![Screenshot_2023-06-09-17-38-37-687-edit_com.android.chrome.jpg](https://github.com/astarub/campus_app/assets/44099069/162ca8ea-706f-49a5-98f2-7e6da5ff9643)